src/lwc/utilInput/utilInput.html

```diff
@@ -74,18 +70,21 @@
         </template>
 
         <template if:true={isLightningSearch}>
-            <c-ge-form-field-lookup id={fieldApiName}
-                                    data-id='inputComponent'
-                                    field-api-name={fieldApiName}
-                                    object-api-name={objectApiName}
-                                    onchange={handleValueChange}
-                                    default-value={fieldValue}
-                                    required={isRequired}
-                                    object-describe-info={objectInfo}
-                                    label={uiLabel}
-                                    variant={variant}
-                                    qa-locator-base={uiLabel}>
-            </c-ge-form-field-lookup>
+            <lightning-record-edit-form object-api-name={objectApiName}>
+                <label class="slds-form-element__label">
```

Comment:
Do we need a `for` attribute here?

Reply:
I don't think we can use it correctly here. It's one of the downsides of using this lightning-input-field component. We want the `for` attribute to point to the id of the `<input>` component, which is inside of `<lightning-input-field/>` and not knowable.

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Form_labelable